WeatherForecast/WeatherForecast/WeatherDetailViewController.swift

```diff
@@ -6,12 +6,22 @@
 
 import UIKit
 
-class WeatherDetailViewController: UIViewController {
-
-    var weatherForecastInfo: WeatherForecastInfo?
-    var cityInfo: City?
-    var tempUnit: TempUnit = .metric
+final class WeatherDetailViewController: UIViewController {
     
+    let iconImageView: UIImageView = UIImageView()
+    let weatherGroupLabel: UILabel = UILabel()
+    let weatherDescriptionLabel: UILabel = UILabel()
+    let temperatureLabel: UILabel = UILabel()
+    let feelsLikeLabel: UILabel = UILabel()
+    let maximumTemperatureLable: UILabel = UILabel()
+    let minimumTemperatureLable: UILabel = UILabel()
+    let popLabel: UILabel = UILabel()
+    let humidityLabel: UILabel = UILabel()
+    let sunriseTimeLabel: UILabel = UILabel()
+    let sunsetTimeLabel: UILabel = UILabel()
+    let spacingView: UIView = UIView()
```

Comment:
1. 접근 제어자가 붙으면 좋을 거 같아요.
2. 아래와 같이 쓰면 조금 더 코드 량이 줄어들거 같네용
```suggestion
    let iconImageView: UIImageView = .init()
```